my-calendar/src/java/org/sakaiproject/widgets/mycalendar/ui/WidgetPage.html

```diff
@@ -13,31 +13,61 @@
 <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script>
 <script type="text/javascript">
 
-// This will change once we make a call to get the events. 
-// At the moment one event will show up in the calendar, for testing purposes only
-var Event = function(text) {
-    this.text = text;
+var Event = function(date, title, type, duration) {
+    this.date = date;
+    this.title = title;
+    this.type = type;
+    this.duration = duration;
 };
-var events = {};
-events[new Date("05/14/2016")] = new Event("Test Event");
+
+jQuery.ajax({
+    url: '/direct/calendar/site/14d3529e-da64-475a-b70f-765c2ed8bb50.json',
```

Comment:
OK for testing but not portable as its a hardcoded siteId. We need to get the siteId. I may just get Wicket to dump the siteId into an element so we can get it.